nab-starter/src/main/java/ru/hh/nab/starter/ConsulService.java

```diff
@@ -138,7 +143,7 @@ public void register() {
 
   private ImmutableRegistration registerWithWeight(int weight) {
     ServiceWeights serviceWeights = ImmutableServiceWeights.builder().passing(weight).warning(weight / warningDivider).build();
```

Comment:
Раз уж тут трогаем поблизости:
```suggestion
    ServiceWeights serviceWeights = ImmutableServiceWeights.builder().passing(weight).warning(0).build();
```
Это какой-то бред, не знаю зачем сделан. Вот тут обсуждали: https://hhru.slack.com/archives/C018CAVU481/p1606919007103900

Reply:
Если так правильно, то во Фронтике я тоже так сделаю.

Reply:
Ага, было бы круто